common/setups/rasr/hybrid_system.py

```diff
@@ -468,7 +469,6 @@ def nn_recog(
                     checkpoints=checkpoints,
                     train_job=train_job,
                     recognition_corpus_key=dev_c,
-                    acoustic_mixture_path=self.train_input_data[train_corpus_key].acoustic_mixtures,
```

Comment:
I think removing this is actually correct, this seems like a bug to me.
But please open a separate PR for this.